ckanext/security/authenticator.py

```diff
@@ -8,10 +8,10 @@
 
 from ckan.lib.authenticator import UsernamePasswordAuthenticator
 from ckan.model import User
-from ckan.common import config
```

Comment:
Why is `config` being removed?
It's still needed to get `ckanext.security.brute_force_key`

Reply:
Though it might make sense to switch to `ckan.plugins.toolkit.config`.